src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/hmi/rc_on_remote_control_settings_notification.cc

```diff
@@ -69,9 +69,6 @@ void RCOnRemoteControlSettingsNotification::DisallowRCFunctionality() {
   for (Apps::iterator it = apps.begin(); it != apps.end(); ++it) {
     application_manager::ApplicationSharedPtr app = *it;
     DCHECK(app);
-    application_manager_.ChangeAppsHMILevel(
```

Comment:
once this is removed, is there any code to free all the allocated module types?

Reply:
@yang1070 
line 119: resource_allocation_manager_.ResetAllAllocations(); -  releases all module types 
https://github.com/smartdevicelink/sdl_core/pull/2197/files/c8e39404819355d48b659d47b5b82cbeaea02d6d#diff-ba35cbefd4f98d7e39ae6651ba039584R119

Reply:
@LitvinenkoIra that looks good to me